lib/index.js

```diff
@@ -47,78 +47,67 @@ module.exports = function loader (css, map) {
 
   validateOptions(require('./options.json'), options, 'PostCSS Loader')
 
-  const rc = {
-    path: path.dirname(file),
-    ctx: {
-      file: {
-        extname: path.extname(file),
-        dirname: path.dirname(file),
-        basename: path.basename(file)
-      },
-      options: {}
+  Promise.resolve().then(() => {
+    if (!options.config || (options.config && !options.config.path)) {
```

Comment:
`postcss-load-config` has it's config lookup algo backed in via cosmiconfig, `options.config.path` should only be used for cases like 

```
|– .config
| |– postcss.config.js
|– src
| |– style.css
|
|– webpack.config.js
```

The current trigger for loading `postcss.config.js` is if no other options are set in `webpack.config.js` =>`postcss.config.js`